pkgs/development/libraries/soundtouch/default.nix

```diff
@@ -1,14 +1,13 @@
-{stdenv, lib, fetchFromGitLab, autoconf, automake, libtool}:
+{ stdenv, lib, fetchgit, autoconf, automake, libtool }:
 
 stdenv.mkDerivation rec {
   pname = "soundtouch";
-  version = "2.2";
+  version = "2.3.1";
 
-  src = fetchFromGitLab {
-    owner = pname;
-    repo = pname;
+  src = fetchgit {
```

Comment:
Please use fetchFromGitea.

Reply:
fixed, thought fetchFromGitea was only for gitea.com, didn’t consult the docs